Adds `phlare.scrape` and `phlare.write` component to support Profiling. (#2552)
grafana/agent: docs/sources/flow/reference/components/phlare.write.md

---
aliases:
- /docs/agent/latest/flow/reference/components/phlare.write
title: phlare.write
---

# phlare.write

> `phlare.write` forward over the network profiles sent from other components to a series of
> user-supplied endpoints. Profiles are sent over the network using [Phlare' Push API](https://grafana.com/oss/phlare/).

Reviewer: ```suggestion
`phlare.write` receives network profiles from other components and forwards them to a series of user-supplied endpoints using [Phlare' Push API](https://grafana.com/oss/phlare/).
```
